services/frontend/pkg/revaconfig/config.go

```diff
@@ -301,3 +317,34 @@ func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error
 		},
 	}, nil
 }
+
+func readMultilineFile(path string) (interface{}, error) {
```

Comment:
Please return a `map[string]struct{}`. Or is there a reason for returning an interface?